### ISSUE TITLE
[Discussion] What's your EXP/hr

### ISSUE THREAD

**Issue by cryzies** [j-e-k/poketrainer#278]
What are you guys pushing for EXP/hr, I'm getting around 70-90k with these settings, would love to see what settings you guys are using and how you achieved your EXP/hr
http://pokerev.r3v3rs3.net/mapui/#15/32.7146/-117.1625

"location": "32.714627, -117.162521",
      "USE_GOOGLE": false,
      "GMAPS_API_KEY": "some api key here",
      "KEEP_CP_OVER": 1800,
      "KEEP_IV_OVER": 95,
      "STEP_SIZE": 100,
      "EXPERIMENTAL": true,
      "SKIP_VISITED_FORT_DURATION": 300,
      "SPIN_ALL_FORTS": true,
      "STAY_WITHIN_PROXIMITY": 2000,
      "AUTO_USE_LUCKY_EGG": false,
      "EGG_INCUBATION": {
        "ENABLE": true,
        "USE_DISPOSABLE_INCUBATORS": true,
        "BIG_EGGS_FIRST": true
      },
      "MIN_ITEMS": {
        "ITEM_POKE_BALL": 30,
        "ITEM_GREAT_BALL": 100,
        "ITEM_ULTRA_BALL": 300,
        "ITEM_POTION": 1,
        "ITEM_SUPER_POTION": 1,
        "ITEM_HYPER_POTION": 1,
        "ITEM_MAX_POTION": 300,
        "ITEM_BLUK_BERRY": 10,
        "ITEM_NANAB_BERRY": 10,
        "ITEM_REVIVE": 1,
        "ITEM_MAX_REVIVE": 50,
        "ITEM_RAZZ_BERRY": 100
      },
      "POKEMON_EVOLUTION": {
        "PIDGEY": 12,
        "WEEDLE": 12,
        "CATERPIE": 12,
        "RATTATA": 25,
        "ABRA": 25,
        "GEODUDE": 25,
        "EKANS": 50,
        "PIKACHU": 50,
        "SANDSHREW": 50,
        "CLEFAIRY": 50,
        "VULPIX": 50,
        "JIGGLYPUFF": 50,
        "ZUBAT": 50,
        "PARAS": 50,
        "VENONAT": 50,
        "DIGLETT": 50,
        "MEOWTH": 50,
        "PSYDUCK": 50,
        "MANKEY": 50,
        "GROWLITHE": 50,
        "TENTACOOL": 50,
        "PONYTA": 50,
        "SLOWPOKE": 50,
        "MAGNEMITE": 50,
        "DODUO": 50,
        "SEEL": 50,
        "GRIMER": 50,
        "SHELLDER": 50,
        "DROWZEE": 50,
        "KRABBY": 50,
        "VOLTORB": 50,
        "EXEGGCUTE": 50,
        "CUBONE": 50,
        "KOFFING": 50,
        "RHYHORN": 50,
        "HORSEA": 50,
        "GOLDEEN": 50,
        "STARYU": 50,
        "OMANYTE": 50,
        "KABUTO": 50
      },
      "MIN_SIMILAR_POKEMON": 0,
      "KEEP_POKEMON_NAMES": ["MEWTWO","SNORLAX","LAPRAS","DRAGONITE","VAPOREON","SLOWBRO","GOLEM","MUK","EXEGGUTOR","CHARIZARD","VENUSAUR","BLASTOISE"],
      "THROW_POKEMON_NAMES": ["PIDGEY"],
      "LIST_POKEMON_BEFORE_CLEANUP": false,
      "LIST_INVENTORY_BEFORE_CLEANUP": false,
      "MAX_CATCH_ATTEMPTS": 10,
      "RELEASE_DUPLICATES": false,
      "RELEASE_DUPLICATES_MAX_LV": 0,
      "RELEASE_DUPLICATES_SCALER": 0,
      "DEFINE_POKEMON_LV": "CP"

**Comment by DannyiCracked** [comment]
https://pkre.slack.com/messages/poketrainer/ we are on slack.

**Comment by mdjong1** [comment]
Yeah..... I don't trust these statistics for shit. But it's what they say...
![screenshot_1](https://cloud.githubusercontent.com/assets/4410453/17266873/27c3806c-55fe-11e6-85a1-b50af4dbb470.png)

**Comment by j-e-k** [comment]
I have no idea how accurate these stats are

**Comment by xezero** [comment]
Something most certainly is off...

```
2016-07-29 17:43:35,581 [    pgoapi] [ INFO] === Exp/Hour: 3978657.51 ===
2016-07-29 17:43:35,605 [    pgoapi] [ INFO] Player Stats: Level: 32, XP: 3579578/3750000, Pokedex: 145, km walked: 203.730300903
2016-07-29 17:43:35,605 [    pgoapi] [ INFO] === Exp/Hour: 3979472.77 ===
```

**Comment by xezero** [comment]
And that's after an hour in already, it started at a ridiculously high number:

```
2016-07-29 16:46:04,296 [    pgoapi] [ INFO] === Exp/Hour: 0.0 ===
2016-07-29 16:46:04,318 [    pgoapi] [ INFO] Player Stats: Level: 32, XP: 3550603/3750000, Pokedex: 145, km walked: 202.647872925
2016-07-29 16:46:04,318 [    pgoapi] [ INFO] === Exp/Hour: 536059401.21 ===
```

**Comment by xezero** [comment]
Found the bug. I'll fix this shortly. Output the variables that were used to determine exp/hour and looks like starting exp got set to 0, which would explain the ridiculously large numbers.

`[ INFO] === Starting Exp: 0.0, Current Exp: 3586983.0, Running Time (hours): 0.010000, Exp/Hour: 280937500.49 ===`

**Comment by xezero** [comment]
Put in a fix for XP calculation in #284

**Comment by j-e-k** [comment]
Great thanks
